### DISCUSSION TITLE
limit or filter HA devices exposed to Alexa

### DISCUSSION THREAD

**Discussion by tapsystem** [alandtse/alexa_media_player#2772] [Q&A] [answered]
Hi,
If i make a good understanding of the doc ( not sure) i can  use the configuration panel to include or exclude alexa devices that are exposed to HA system, but how can i filter / declare HA devices that i want or not to be accessible by Alexa ? As example i want to make all alarm  entries invisible by Alexa because i dont want alexa to be able to manage alarm.

Thanks by advance

**Comment by danielbrunt57** [comment, marked as answer]
Alexa Media Player (alexa_media) has nothing to do with [Alexa via Home Assistant Cloud](https://www.home-assistant.io/cloud/alexa/) / [Alexa Smart Home Skill](https://www.home-assistant.io/integrations/alexa.smart_home/) which make HA entities available to Alexa.

If you want to exclude any alarm entries you already have in HA from being made available to Alexa, you need to look at your HA Cloud setup for Alexa & it's UI for  including/excluding devices, or, if you've configured the skill manually, your `alexa:` `smart_home:` entry in `configuration.yaml`.

**Reply by tapsystem** [reply]
Ohhhh that why i can got it working ... 😁🤯
Thanks a lot to put me on the good way ! 
And thanks for the great job !
